handbook/product/design/design_process.md

```diff
@@ -98,7 +98,12 @@ While asynchronous communication is a core attribute of remote work, key moments
 
 - Visual design should utilize the Sourcegraph’s Figma based component system
 - Dark compositions will be created for each major design
-- If new components are required, components will be created in the file itself, not the component system
+- If new components are required, the following process should be followed
+    - Design component in the Figma file which requires it
+    - @ mention a frontend engineer requesting an addition or change to the component system
+        - Engineering approves, requests changes, or requests the designer uses an existing component
+    - @ mention a designer and request a review
```

Comment:
We were using Chromatic for this when working with @felixfbecker. I think we should add this detail to the guidelines so it's clear that this is the correct path.